### PR TITLE
fix(Plex): Cover Images

### PR DESCRIPTION
**Pull request by TheVisual**
## Description 
Fix Plex Cover Art
Resolves https://github.com/PreMiD/Presences/issues/7337

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
image

</details>

### CODE REVIEW

**Comment by DarkVillager** [comment]
@TheVisual bump
version in metadata.json

File: websites/P/Plex/presence.ts [hunk 1]
```diff
@@ -136,25 +136,28 @@ function getTranslation(stringName: string): string {
...
 	isUploading = true;
 
-	const file = await fetch(url).then(x => x.arrayBuffer()),
-		outputUrl = await fetch("https://bashupload.com", {
-			method: "POST",
-			body: file,
-		})
-			.then(x => x.text())
-			.then(x => x.match(/https(.*)/)?.[0]);
+	const file = await fetch(urltoupload).then(x => x.blob()), // Fetch the file as a Blob object
```
Comment (theusaf) [inline websites/P/Plex/presence.ts:145]: These comments are redundant, and just repeat what the code is doing.
```suggestion
	const file = await fetch(urltoupload).then(x => x.blob()),
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by TheVisual** [comment]
@DarkVillager @theusaf i have applied your suggestions.

**Comment by DarkVillager** [comment]
Resolves #7337

**Comment by TheVisual** [comment]
@DarkVillager done

File: websites/P/Plex/presence.ts [hunk 2]
```diff
@@ -136,25 +136,27 @@ function getTranslation(stringName: string): string {
 let isUploading = false;
 
 const uploadedImages: Record<string, string> = {};
-async function uploadImage(url: string): Promise<string> {
+async function uploadImage(urltoupload: string): Promise<string> {
```
Comment (theusaf) [inline websites/P/Plex/presence.ts:139]: For consistent variable naming, something like `url` or `urlToUpload` or `uploadUrl` may be better.

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/P/Plex/presence.ts (hunk 2 above)
Comment (TheVisual) [reply websites/P/Plex/presence.ts:139]: i'll apply these tomorrow it was mothers day

Comment (theusaf) [reply websites/P/Plex/presence.ts:139]: This doesn't appear to have been resolved.

**Review by Bas950 — CHANGES_REQUESTED** [review]
Please use PD
You can now use POST requests to /create/base64 with text/plain
or a POST to /create/image with multipart file: <file>

**Comment by Bas950** [comment]
Converted to draft for now, we need to fix a few things in PD first.

**Comment by theusaf** [comment]
Issues with PD have been resolved.

**Pull request closed without merging**